src/components/Classes/components/JoinClassModal.js

```diff
@@ -1,89 +1,107 @@
-import React from "react";
-import * as fetch from "../../../lib/fetch.js";
-import { Redirect } from "react-router-dom";
-
-import { Button, Container, Row, Col, FormGroup, Label, Input } from "reactstrap";
-
-import ReactModal from "react-modal";
-
-class JoinClassModal extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      wid: "",
-      disableSubmit: false,
-      error: "",
-      redirect: false,
-    };
-  }
-
-  closeModal = () => {
-    if (this.props.onClose && {}.toString.call(this.props.onClose) === "[object Function]") {
-      this.props.onClose();
+import React, { useState } from 'react';
+// import * as fetch from "../../../lib/fetch.js";
```

Comment:
hm this doesn't look right, probably shouldn't be commented out. I'd assume this is what's causing https://gyazo.com/15dcc9b74c72bea286e9330f57b136d0

Reply:
with or without it still causes the problem in the ss D:
And when I try to run through ESlint with "import * as fetch from "../../../lib/fetch.js";", it throws an error

Reply:
Is this locally? And how are you running the editor? It looks like the backend might not be up; try `npm run staging_start`

Reply:
Yea I tested it, and with it, it works for me

Reply:
npm run staging_start works for me as well, so i uncommented it now